beta/src/components/MDX/TerminalBlock.tsx

```diff
@@ -28,8 +28,7 @@ function TerminalBlock({level = 'info', children}: TerminalBlockProps) {
   if (typeof children === 'string') {
     message = children;
   } else if (
-    React.isValidElement(children) &&
-    typeof children.props.children === 'string'
+    React.isValidElement(children) 
```

Comment:
Do we understand why this transformation exists at all? What happens if you remove it altogether?

Reply:
I hope the condition was added to render the strings directly and render the children if it has collection of elements together (Like the commands in two lines).

This check can be removed and we can render the children of terminalBlocks directly too. But when we render the children directly the texts are wrapped inside `<p>` and takes extra space like this.
<img width="797" alt="Screenshot 2021-11-05 at 12 27 54 AM" src="https://user-images.githubusercontent.com/32865581/140402730-b215b5e4-8393-45c9-aebe-017fb9650024.png">